en/advanced_config/bootloader_update.md

```diff
@@ -128,20 +136,21 @@ The following steps explain how you can "manually" update the bootloader using t
    Reading symbols from px4fmuv5_bl.elf...done.
    ```
 
-1. Find your `<dronecode-probe-id>` by running an ls command in the **/dev/serial/by-id** directory.
-1. Now connect to the Dronecode probe with the following command:
+1. Find your `<dronecode-probe-id>` by running an `ls` command in the **/dev/serial/by-id** directory.
+
+1. Now connect to the debug probe with the following command:
 
    ```sh
    tar ext /dev/serial/by-id/<dronecode-probe-id>
    ```
 
-1. Power on the Pixhawk with another USB cable and connect the Dronecode probe to the FMU-DEBUG port.
+1. Power on the Pixhawk with another USB cable and connect the probe to the `FMU-DEBUG` port.
```

Comment:
Assumption here is all probes connect to this port.

Reply:
Yes correct